drivers/volume/portworx/portworx.go

```diff
@@ -360,10 +362,26 @@ func (p *portworx) OwnsPVC(pvc *v1.PersistentVolumeClaim) bool {
 	} else {
 		storageClass, err := k8s.Instance().GetStorageClass(storageClassName)
 		if err != nil {
```

Comment:
For pre-provisioned volumes created using https://docs.portworx.com/portworx-install-with-kubernetes/storage-operations/create-pvcs/using-preprovisioned-volumes/#using-the-portworx-volume

Wouldn't `OwnsPVC` returns from line 354 itself?

I have a statically provisioned PVC using PX that doesn't have annotations and a storage class. It only has pv.Spec.PortworxVolume.

Reply:
I was reproing similar to what a user had hit where they were populating a random UUID for the storageclass. Will fix for the above case too

Reply:
Updated